file3.txt

```diff
@@ -1,6 +1 @@
-data by ahmad fraz
-andrew <i class="fas fa-table-tennis fa-xs fa-fw"></i>
-added <i class="fa fa-html5" aria-hidden="true"></i>
-alpha bravo charlie
-mad max fury
-bad boys
+hello
```

Comment:
this change is concise and short but its better to add meaning full comments for future .